tests/backup/backup_orphaned_test.go

```diff
@@ -424,12 +424,12 @@ var _ = Describe("{DeleteUserBackupsAndRestoresOfDeletedAndInActiveClusterFromAd
 					nonAdminCtx, err := backup.GetNonAdminCtx(user, commonPassword)
 					log.FailOnError(err, "failed to fetch user %s ctx", user)
 					for _, provider := range providers {
-						userCloudCredentialName := fmt.Sprintf("autogenerated-cred-%v", time.Now().Unix())
+						userCloudCredentialName := fmt.Sprintf("autogenerated-cred-%v", RandomString(5))
```

Comment:
Should we add the user info a as well and reduce the random string to make it easy to trace and track if the testcases fail again?

Reply:
Added non-admin user name for all objects.